contracts/tests/psp721.rs

```diff
@@ -129,18 +166,6 @@ mod psp721 {
         );
         // Token Id 2 does not exists.
         assert_eq!(nft.owner_of([1; 32]), None);
-        // Create token Id 2.
-        assert!(nft._mint([1; 32]).is_ok());
-        // Alice owns 1 token.
-        assert_eq!(nft.balance_of(accounts.alice), 1);
-        // Token Id 2 is owned by Alice.
-        assert_eq!(nft.owner_of([1; 32]), Some(accounts.alice));
-        change_caller(accounts.bob);
-        // Bob cannot transfer not owned tokens.
-        assert_eq!(
```

Comment:
You removed useful test=) Return it back, please + split that one test into two tests